src/common/meta/src/rpc/ddl.rs

```diff
@@ -176,6 +179,7 @@ pub struct DropTableTask {
     pub schema: String,
     pub table: String,
     pub table_id: TableId,
+    pub drop_if_exists: bool,
```

Comment:
For backward compatibility.
```suggestion
    #[serde(default)]
    pub drop_if_exists: bool,
```

Reply:
Updated.